hermes/file_gen_test.go

```diff
@@ -47,15 +50,15 @@ func TestChecksum(t *testing.T) {
 	}
 }
 
-// Test function for the generateFileName function in file_gen.go
+// TestFileName tests the generateFileName method.
```

Comment:
It's not strictly required to comment exported functions in _test.go files.
Only add comments where the name is not self-describing or something unusual/complicated is going on.

Reply:
Would it be better to not export them then? Will delete unnecessary comments!

Reply:
In this case, you need to export them since these are your test functions. If you don't then all your tests will pass, but none will actually run. Just remove the comments.

Reply:
@alicjakwie is making these changes in a different PR: https://github.com/googleinterns/step224-2020/pull/58 so I'll mark this resolved for now.